front/style/_forms.scss

```diff
@@ -45,13 +45,29 @@
         margin-bottom: 15px;
     }
 
+    //&__blue_row {
```

Comment:
This code is a question (I forgot to ask) because I could not handle blue fields like in your mockup.

So it's up to you, either I leave it as is or just remove it, but I won't go further on the integration work.